packages/twenty-front/src/modules/views/components/ViewBarDetails.tsx

```diff
@@ -29,21 +29,23 @@ const StyledBar = styled.div`
   border-top: 1px solid ${({ theme }) => theme.border.color.light};
   display: flex;
   flex-direction: row;
-  min-height: 32px;
+  align-items: center;
   justify-content: space-between;
+  position: relative;
+  padding: ${({ theme }) => theme.spacing(1)};
   z-index: 4;
-  padding-top: ${({ theme }) => theme.spacing(1)};
-  padding-bottom: ${({ theme }) => theme.spacing(1)};
+  min-height: 32px;
 `;
 
 const StyledChipcontainer = styled.div`
   align-items: center;
   display: flex;
   flex-direction: row;
   gap: ${({ theme }) => theme.spacing(1)};
-  min-height: 32px;
-  margin-left: ${({ theme }) => theme.spacing(2)};
-  flex-wrap: wrap;
+  overflow-x: scroll;
+  padding-top: ${({ theme }) => theme.spacing(1)};
+  white-space: nowrap;
```

Comment:
let's modify the SortOrFilterChip itself and add this white-space: no-wrap property there